Disable "Download Page as CSV" button if layout does not support it (#22281)
directus/directus: .changeset/big-kiwis-happen.md

---
'@directus/app': patch
---

> Hide "Download Page as CSV" button if the selected layout does not support his action

Reviewer: ```suggestion
Ensured "Download Page as CSV" button is disabled if the selected layout does not support that action
```
